### DISCUSSION TITLE
简直就是我的梦中情"note"（还有一些使用过程中发现的bug）

### DISCUSSION THREAD

**Discussion by zslxiu** [tamlok/vnote#2447] [General]
因为想把数据完完全全的掌握在自己手里，最近一直在离线笔记软件，对比了很多款，vnote真的好用，简约易用，可以切换阅读模式，导出的文件内容相比于其他软件都更好。感谢作者开发出这样的笔记软件。

使用过程中发现了一些bug：
1、我在自己的笔记本上（win11 21h2）使用全屏模式发现在左侧导航栏无法右键呼出菜单，在编辑页面也无法呼出菜单，且打开新的笔记会闪烁，使用win10 22h2系统全屏模式右键可以呼出菜单但是闪烁问题还是有，包括右键菜单也会闪烁。（个人感觉全屏模式其实不是很有必要😄）
2、如果拖动侧边栏（包括大纲、已打开窗口、导航等）往您预设的上、下、右三个预留的边栏位置放，但是不放入的话，就会导致界面出现一条线（就是比如说想将右侧大纲栏调到上方的位置，原来的整个界面会往下调整留出位置的那条线），重开软件才会消失。
3、最小化到托盘偶现过界面整体异常的问题

**Comment by tamlok** [comment]
Hi, welcome to VNote!

Please try setting openGL option to solve your issues. See https://github.com/vnotex/vnote/issues/853

**Reply by zslxiu** [reply]
Thanks
